test/Files.test.js

```diff
@@ -73,6 +79,11 @@ describe('missing implementation', () => {
   test('_getPresignUrl', async () => {
     await expect(files._getPresignUrl('anything')).rejects.toThrow('not implemented')
   })
+  test('_initWithNewCreds', async () => {
+    initWithNewCredsMock.mockRestore()
```

Comment:
same here?

Reply:
This one is needed as we want to test the actual File.js impl instead of the mock. So its restored to actual impl.